CMakeLists.txt

```diff
@@ -100,16 +100,19 @@ else()
     message(FATAL_ERROR "Unknown OS type")
 endif()
 
-if(NOT DEFINED UMF_HWLOC_REPO)
-    set(UMF_HWLOC_REPO "https://github.com/open-mpi/hwloc.git")
-endif()
+if(UMF_DISABLE_HWLOC)
+    message(WARNING "hwloc is disabled, hence OS provider, memtargets, "
```

Comment:
hmm... I'm not sure if this should be a warning - this option is used in llvm project now, so they will see this warning all the time, right...?

Reply:
I think that STATUS suffices just to notify the user. We shouldn't warn on using an option that is legitimate.

Reply:
yeah, but on the other hand it's disabling half of our functionality 😉

Reply:
I've changed to STATUS